pkgs/development/libraries/lib2geom/default.nix

```
@@ -0,0 +1,31 @@
+{ stdenv, fetchurl, cmake, ninja, pkgconfig, boost, gsl, gtk2, gtkmm2, cairomm, cairo, gtest }:
+
+stdenv.mkDerivation rec {
+  name = "lib2geom-1.0.0";
+
+  outputs = [ "out" "dev" ];
+
+  src = fetchurl {
+    url = "http://alpha.inkscape.org/releases/${name}.tar.bz2";
+    sha256 = "0hzzx44c7rv5fwzcd9qgggjkk14y2k7g6y52cqsnvl7avr5balci";
+  };
+
+  nativeBuildInputs = [ cmake ninja pkgconfig ];
+  buildInputs = [ boost gsl gtk2 gtkmm2 cairomm cairo ];
+  checkInputs = [ gtest ];
+
+  doCheck = true;
+
+  # TODO: move this into ninja setup-hook
+  checkPhase = ''
+    ninja test
+  '';
+
+  meta = with stdenv.lib; { 
+    description = "Easy to use 2D geometry library in C++";
+    homepage = https://gitlab.com/inkscape/lib2geom;
+    license = [ licenses.lgpl21Plus licenses.mpl11 ];
```

```suggestion
    license = [ licenses.lgpl21 licenses.mpl11 ];
```
Headers in source files don't seem to mention "or any later version" for the LGPL

---

Thank you.